technique/muon/zero_field_setup.py

```diff
@@ -0,0 +1,278 @@
+from genie_python import genie as g
+import time
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy import stats
+from builtins import range
+from builtins import input
+
+PV_AUTO_FEEDBACK_MODE = "TE:NDLT1205:ZFCNTRL_01:AUTOFEEDBACK"
+PV_MAGNETIC_FIELD_STRENGTH = "TE:NDLT1205:ZFMAGFLD_01:FIELDSTRENGTH"
+
+PV_SETPOINT_X = "TE:NDLT1205:ZFCNTRL_01:FIELD:X:SP"
+PV_SETPOINT_Y = "TE:NDLT1205:ZFCNTRL_01:FIELD:Y:SP"
+PV_SETPOINT_Z = "TE:NDLT1205:ZFCNTRL_01:FIELD:Z:SP"
+
+POWER_SUPPLY_MAX = 100.00
```

Comment:
`POWER_SUPPLY_MAX` is actually the difference between the lower and upper limits for the current. You can get these from `ZFCNTRL_01:OUTPUT:X:CURR:SP.DRVL` and `ZFCNTRL_01:OUTPUT:X:CURR:SP.DRVH`

Reply:
Done.